templates/Makefile.in

```diff
@@ -22,11 +23,13 @@ test:
 .PHONY: format
 format:
 	python -m ufmt format $(SOURCES)
```

Comment:
If you have time, can you either

1. make ufmt dep use `ufmt[ruff]` and configure to use [ruff-api](https://ufmt.omnilib.dev/en/stable/guide.html#formatter)
2. remove ufmt entirely, and just use ruff for import sorting as well as formatting (needs `"I"` added to the extend-select at a minimum)